src/api_manager/api_manager_impl.cc

```diff
@@ -319,9 +323,6 @@ bool ApiManagerImpl::ReWriteURL(const char *uri, const size_t uri_len,
 
 std::unique_ptr<RequestHandlerInterface> ApiManagerImpl::CreateRequestHandler(
     std::unique_ptr<Request> request_data) {
-  if (config_manager_) {
-    config_manager_->CountRequests(1);
-  }
```

Comment:
Why delete this?  Does that mean ESP still auto update, even no traffic?

Reply:
The old code is using every minute timer.  When a timer is fired,  use this one to check if there are any traffic,  if no traffic, do nothing.

The new code is not using periodical timer.  It purely uses the rollot_id from Check/Report response.   If rollot_id is changed,  fires a randomly throttled timer to fetch new service config.

If there is not traffic, will be no Check/Report response,  timer will not be fired.   So this code is not needed any more.